README.md

```diff
@@ -79,20 +79,14 @@ This module targets Foreman 2.4+.
 ## Types and providers
 
 `foreman_config_entry` can be used to manage settings in Foreman's database, as
-seen in _Administer > Settings_. Provides:
+seen in _Administer > Settings_.
 
-* `cli` provider uses `foreman-rake` to change settings (default)
-
-`foreman_hostgroup` can create and manage host group in Foreman's database.
-Providers:
-
-* `rest_v2` provider uses API v2 with apipie-bindings and OAuth (default)
+* `cli` provider uses `foreman-rake` to change settings
```

Comment:
How about making it a nice flowing paragraph now that it's no longer a list?
```suggestion
seen in _Administer > Settings_. The `cli` provider uses `foreman-rake` to change settings.
```

Probably the same for `foreman_smartproxy`

Reply:
Updated. (I thought I'd pushed last night, but there'd been some network error and it hadn't made it out)